### DISCUSSION TITLE
help!! the output openMVS scene file cannot be written

### DISCUSSION THREAD

**Discussion by YoungJackson** [cdcseacave/openMVS#669] [Q&A]
when I use openMVG2openMVS ，It cannot run 
![Snipaste_2021-04-08_19-50-20](https://user-images.githubusercontent.com/43715415/114021857-ba712300-98a3-11eb-95b5-ca864ba5e982.png)

**Comment by cdcseacave** [comment]
the error message is quite clear, it can not find the original images; probably you moved them, etc
plus this is an OpenMVG application, pls post there

**Reply by YoungJackson** [reply]
OK，thanks, I can use it two days ago , but today it didn't work, 
I will go to the openMVG
